bcda/auth/client/ssas_test.go

```diff
@@ -105,200 +158,163 @@ func (s *SSASClientTestSuite) TestNewSSASClient_TLSFalseNoURL() {
 	assert.EqualError(s.T(), err, "SSAS client could not be created: no URL provided")
```

Comment:
separate from the work in this PR, but it might be useful to update this error message to something like:

`SSAS client could not be created: no certificate provided`